### PR TITLE
Bump consistent eval to 0.0.50

### PR DESCRIPTION
**Pull request by jeroach**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1156,9 +1156,9 @@
       }
     },
     "@brightspace-ui-labs/list-item-accumulator": {
-      "version": "1.0.1",
-      "resolved": "https://registry.npmjs.org/@brightspace-ui-labs/list-item-accumulator/-/list-item-accumulator-1.0.1.tgz",
-      "integrity": "sha512-85ae9qGkFtaDasFjKtcdnhvF2gl+eUd133BVR44QmWE3iped1ZO3p5gk+UCuaIXqM5jxFthoQnjYrR9+A7G6LQ==",
+      "version": "1.0.2",
+      "resolved": "https://registry.npmjs.org/@brightspace-ui-labs/list-item-accumulator/-/list-item-accumulator-1.0.2.tgz",
+      "integrity": "sha512-6XjgEP4rU2htngpn37fCN7tLkhhaxbelmy671oUSLjpmZ0zbsR5taZc4TZniKA+bZKhw272y89LuldTz32rjLw==",
```
Comment (jeroach) [inline package-lock.json:1161]: fyi @zinabat

File: package-lock.json [hunk 2]
```diff
@@ -1209,9 +1209,9 @@
       }
     },
     "@brightspace-ui/core": {
-      "version": "1.68.0",
-      "resolved": "https://registry.npmjs.org/@brightspace-ui/core/-/core-1.68.0.tgz",
-      "integrity": "sha512-0ks1yh+fjEB0edhbylTcfCPuF2fCHfFzI464uMNrXvUZ+Dbw3qFota1c66DI1+Imls4yh4JOdxJkOy9GNfkHEw==",
+      "version": "1.70.0",
+      "resolved": "https://registry.npmjs.org/@brightspace-ui/core/-/core-1.70.0.tgz",
+      "integrity": "sha512-3/su4EfU2zauv4fxv6qQxyRiD+ql1osQpgcJl82obeShE/LhO0Ioav6nMoGIeCW2SMEiG3rKQ0MGSrLVYajYqw==",
```
Comment (jeroach) [inline package-lock.json:1214]: fyi @margaree

File: package-lock.json [hunk 3]
```diff
@@ -4496,13 +4506,13 @@
       }
     },
     "d2l-consistent-evaluation": {
-      "version": "github:BrightspaceHypermediaComponents/consistent-evaluation#ed8d74336d588acdd801eea3c780690d41163212",
+      "version": "github:BrightspaceHypermediaComponents/consistent-evaluation#f88bd7d090be022a7f0d1945002601c2def0b898",
```
Comment (jeroach) [inline package-lock.json:4509]: fyi @JamieTran

File: package-lock.json [hunk 4]
```diff
@@ -4486,7 +4496,7 @@
       }
     },
     "d2l-common": {
-      "version": "github:BrightspaceHypermediaComponents/common#ce96849af3ab72a2882536a0bd015f009fb64372",
+      "version": "github:BrightspaceHypermediaComponents/common#8506497b994c3e7ffdd14fc5ccabbbefd20bff00",
```
Comment (jeroach) [inline package-lock.json:4499]: fyi @svanherk

File: package-lock.json [hunk 5]
```diff
@@ -4633,19 +4643,6 @@
         "@polymer/polymer": "^3"
       }
     },
-    "d2l-engagement-dashboard": {
```
Comment (jeroach) [inline package-lock.json:4636]: Hmmm, not sure why this was dropped, but it seems to be failing the CI.

Comment (svanherk) [reply package-lock.json:4636]: I was also seeing this with my `npm i` installs, and @dlockhart wasn't.  When I went to his versions it stopped:
`node`: `12.18.2`
`npm`: `6.14.5`

Comment (jeroach) [reply package-lock.json:4636]: thanks, I'll try that

Comment (jeroach) [reply package-lock.json]: fixed it - thanks a bunch @svanherk :)

**Pull request merged**